takwimu/static/js/charts.js

```diff
@@ -1068,6 +1072,10 @@ function Chart(options) {
 
     chart.actionDownload = chart.actionLinks
         .append("a")
+        .attr("ga-on", "click")
+        .attr("ga-event-category", "Data")
+        .attr("ga-event-action","Download")
+        .attr("ga-event-label", chart.id)
```

Comment:
QQ: Does `chart.id` included country? It is important to know which country the data belongs to since it's possible for to data charts to have the exact same name.

Reply:
no they don't, will append the `window.takwimu.country.slug`